### PR TITLE
Add client side validation

### PR DESCRIPTION
**Pull request by alef-garrido**
Added email input validation.

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] it looks like the validation doesn't work. I tried this text and the form were send (check your email, if the form is correct you could see my tests there)
image
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: myfunctions.js [hunk 1]
```diff
@@ -198,9 +199,12 @@ const checkEmail = () => {
...
 };
 
-inputEmail.addEventListener('input', checkEmail);
-contactForm.addEventListener('submit', (event) => {
-  if (!checkEmail) {
-    event.preventDefault();
+const formSubmit = (e) => {
+  if (!checkEmail()) {
+    e.preventDefault();
+    inputEmail.classList.add('error')
```
Comment (Dipeshtwis) [inline myfunctions.js]: - [x] If a user submits the contact form with the email field containing uppercase characters an error message should show up. (for now it's validating and making input as red, but don't you think it's better to give error message as well like `email should be in lowercase` for recruiter to know why this form has not submitted).
- [ ] The error message should show up close to the "submit" button.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @alef-garrido, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hello @alef-garrido ✋

## STATUS: APPROVED✔️ ✔️
Awesome job on implementing the changes requested by the previous reviewer. You can now merge and proceed to the next project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ All the best!!


Happy coding 💻

**Pull request merged**